README.md

```diff
@@ -45,70 +45,75 @@ The following packages are required for setting up the development environment,
  - `curl`
 
 ## Home Device and Account Manager
-Mycroft AI, Inc. - the company behind Mycroft maintains the Home device and account management system. Developers can sign up at https://home.mycroft.ai
+Mycroft AI, Inc. maintains a device and account management system known as Mycroft Home. Developers may sign up at: https://home.mycroft.ai
 
-By default the Mycroft software is configured to use Home, upon any request such as "Hey Mycroft, what is the weather?", you will be informed that you need to pair and Mycroft will speak a 6-digit code, which you enter into the pairing page on the [Home site](https://home.mycroft.ai).
+By default, Mycroft software is configured to use Home. Upon any request such as "Hey Mycroft, what is the weather?", you will be informed that your device needs to be paired. Mycroft will speak a 6-digit code, which is entered into the pairing page within the [Mycroft Home site](https://home.mycroft.ai).
 
-Once signed and a device is paired, the unit will use our API keys for services, such as the STT (Speech-to-Text) API. It also allows you to use our API keys for weather, Wolfram-Alpha, and various other skills.
+Once signed and paired, your unit will use Mycroft API keys for services such as STT (Speech-to-Text), weather, Wolfram-Alpha, and various other skills.
 
 Pairing information generated by registering with Home is stored in:
 
 `~/.mycroft/identity/identity2.json` <b><-- DO NOT SHARE THIS WITH OTHERS!</b>
 
-It's useful to know the location of the identity file when troubleshooting device pairing issues.
+It is useful to know the location of this identity file when troubleshooting any device pairing issues.
 
-## Using Mycroft without Home.
-If you do not wish to use our service, you may insert your own API keys into the configuration files listed below in <b>configuration</b>.
+## Using Mycroft Without Home.
+If you do not wish to use the Mycroft Home service, you may insert your own API keys into the configuration files listed below in <b>configuration</b>.
 
 The place to insert the API key looks like the following:
 
 `[WeatherSkill]`
 
 `api_key = ""`
 
-Put the relevant key in between the quotes and Mycroft Core should begin to use the key immediately.
+Put a relevant key inside the quotes and Mycroft Core should begin to use the key immediately.
 
-### API Key services
+### API Key Services
+These are the keys currently used in Mycroft Core:
 
 - [STT API, Google STT](http://www.chromium.org/developers/how-tos/api-keys)
 - [Weather Skill API, OpenWeatherMap](http://openweathermap.org/api)
 - [Wolfram-Alpha Skill](http://products.wolframalpha.com/api/)
 
-These are the keys currently in use in Mycroft Core.
-
 ## Configuration
-Mycroft configuration consists of 3 possible config files.
+Mycroft configuration consists of 3 possible files:
 - `mycroft-core/mycroft/configuration/mycroft.conf`
```

Comment:
Since you are changing this, go ahead and add a reference to the remote configuration from home.mycroft.ai.  This is a 4th item that gets applied after the first (aka the defaults).   I have use the terms:  Defaults, Remote, Machine, User to describe these four.